VersionControlUI/src/DiffManager.cpp

```diff
@@ -94,9 +94,11 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 
 	// load newer version
 	diffSetup.newVersionManager_ = createTreeManagerFromVersion(diffSetup.repository_, diffSetup.newVersion_);
+	diffSetup.newVersionManager_->setRevisionName("new");
```

Comment:
no need to change `revision` to `hash` or anything else inside `TreeManager` I like `revisionName` there. I'm just curious, if you have only 'new' and 'old' here, could you save reviews for two commits? Is that somehow independent? If so then we can leave new and old.